_sass/design-system/patterns/_tooltip.scss

```diff
@@ -155,3 +164,7 @@ BEACON
 .ds-beacon-cyan:before{
```

Comment:
In SASS, you can nest styles to keep the code a more readable.  SASS will process nested rules into separate rules in the compiled css file.

In this case, take a look at the first .ds-beacon-cyan rule where you set the background and box shadow.  Since you're modifying that class with a ":before", you can just nest this :before into the first rule.

In other words, you can remove the separate rule  for .de-beacon-cyan:before and combine it with the first one above to make it look like this:

.ds-beacon-cyan {
  background-color:$cyan100; 
  box-shadow:0px 0px 2px 2px $cyan500;

  &:before {
    box-shadow:0px 0px 1px 1px $cyan900;
  }

}

SASS will output this as two separate rules -- one for .ds-beacon-cyan, and one for .ds-beacon-cyan:before